app/src/renderer/components/governance/LiProposal.vue

```diff
@@ -26,8 +31,7 @@ export default {
       if (this.proposal.proposal_status === `Passed`)
         return {
           button: null,
-          message: `This proposal has passed`,
-          color: `green`
```

Comment:
I think we should leave it green here

Reply:
it's not a little green dot anymore - it's a green checkmark.

if felt the green status dot should be for signalling "voting"